### PR TITLE
[NTOS:IO] Move device node functions from pnpmgr.c to devnode.c

### PR DESCRIPTION
**Pull request by Extravert-ir**

### CODE REVIEW

File: ntoskrnl/io/pnpmgr/devnode.c [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+ *              2007 Hervé Poussineau (hpoussin@reactos.org)
+ *              2010-2012 Cameron Gutman (cameron.gutman@reactos.org)
+ */
+
+
+#include <ntoskrnl.h>
+#define NDEBUG
+#include <debug.h>
+
+PDEVICE_NODE IopRootDeviceNode;
```
Comment (ThFabba) [inline ntoskrnl/io/pnpmgr/devnode.c:15]: Wow, kernel source code looks weird without the `/* FUNCTIONS ***********/` thing :D

Comment (Extravert-ir) [reply ntoskrnl/io/pnpmgr/devnode.c:15]: If we want this back, good time to do it :D
But honestly I haven't made my mind about these headers

Comment (ThFabba) [reply ntoskrnl/io/pnpmgr/devnode.c:15]: Yeah I guess I'd prefer adding these comments here for consistency with the rest of the kernel. Less surprise when reading the code = better ;)

Comment (Extravert-ir) [reply ntoskrnl/io/pnpmgr/devnode.c:15]: What is the difference between `GLOBALS` and `DATA`?

Comment (ThFabba) [reply ntoskrnl/io/pnpmgr/devnode.c:15]: I don't think there is a difference. GLOBALS seems more common in ntoskrnl *shrug*

File: ntoskrnl/io/pnpmgr/devnode.c [hunk 2]
```diff
@@ -0,0 +1,400 @@
+/*
```
Comment (ThFabba) [inline ntoskrnl/io/pnpmgr/devnode.c:1]: Diff of "just move all the functions to this file" -> what's actually in the file:
https://gist.github.com/ThFabba/1cedf75456857db56d94cd99c05081f8

Comment (Extravert-ir) [reply ntoskrnl/io/pnpmgr/devnode.c:1]: I indicated this in the commit message!

**Pull request merged**